armhf-hassio-addon/run.sh

```diff
@@ -10,7 +10,7 @@ HASSIO_DEBUG_OTAU="$(jq --raw-output '.debug_otau' $CONFIG_PATH)"
 HASSIO_DECONZ_WEB_PORT="$(jq --raw-output '.web_port' $CONFIG_PATH)"
 HASSIO_DECONZ_WS_PORT="$(jq --raw-output '.websockets_port' $CONFIG_PATH)"
 HASSIO_DECONZ_DEVICE="$(jq --raw-output '.deconz_device' $CONFIG_PATH)"
-HASSIO_DECONZ_VNC_MODE=":$(jq --raw-output '.vnc.active' $CONFIG_PATH)"
+HASSIO_DECONZ_VNC_MODE="$(jq --raw-output '.vnc.active' $CONFIG_PATH)"
 HASSIO_DECONZ_VNC_DISPLAY=":$(jq --raw-output '.vnc.port - 5900' $CONFIG_PATH)"
```

Comment:
what did the semicolon do? Should you remove it for the HASSIO_DECONZ_VNC_DISPLAY value aswell?

Reply:
No, it is correct there. 
The VNC server is started with `tigervncserver :0` and `export DISPLAY=:0` (for port 5900)